tests/runner.py

```diff
@@ -1354,30 +1354,41 @@ class BrowserCore(RunnerCore):
   def __init__(self, *args, **kwargs):
     super(BrowserCore, self).__init__(*args, **kwargs)
 
+  @staticmethod
+  def browser_open(url):
+    if not EMTEST_BROWSER:
+      logger.info('Using default system browser')
+      webbrowser.open_new(url)
+      return
+
+    browser_args = shlex.split(EMTEST_BROWSER)
+    # If the given browser is a string with no separators, assume it's one
+    # of the types that the webbrowser module accepts like `safari`,
+    # `firefox`, or `chrome`. See https://docs.python.org/2/library/webbrowser.html
+    # for the full list
+    if len(browser_args) == 1 and os.path.sep not in browser_args[0]:
```

Comment:
hmm, i realize this would break my own workflow actually, of doing `EMTEST_BROWSER=google-chrome-unstable` or `-stable` etc. That is, those are in the path, but they are probably not recognized by `webbrowser.get`?

Reply:
Gotcha -- reworked it to account for that. Implementation's a little better now :)